src/src/CMakeLists.txt

```diff
@@ -127,12 +127,15 @@ target_link_libraries(kmeans_lib INTERFACE
 #---[ Demos ]------------------------------------------------------------
 add_executable(flat_l2 flat_l2.cc)
 target_link_libraries(flat_l2 PUBLIC kmeans_lib nlohmann_json::nlohmann_json)
+target_compile_definitions(flat_l2 PUBLIC TILEDBVS_ENABLE_STATS)
```

Comment:
Do we do the right thing somewhere in terms of downloading and installing `nlohmann_json`?  I had that in a local subdirectory (with the single-header `json.hpp`).  Do we want to just keeping using it that way for now?  I deleted it in my branch when I excised `json` but we can put it back.  My own preference would be to (eventually) be clean and use it as an external package, but that is not a high priority at all.

Reply:
We keep a local copy of `json.hpp` [like the Core](https://github.com/TileDB-Inc/TileDB/blob/dev/external/include/nlohmann/json.hpp).

We can properly add it as a dependency at a later time (preferably with vcpkg).